.nvmrc

```diff
@@ -1 +1 @@
-10.15.1
+16.15.0
```

Comment:
Isn't the upgrade to 16.14.2 ?

Reply:
When I started this, 16.14.2 was the latest. 16.15 was released on the 26th of April.

So yeah please go ahead and use that.